src/Common/Utils/Net.ts

```diff
@@ -0,0 +1,11 @@
+﻿import * as express from "express";
+
+export module Net {
+    export function sendResponse(response: express.Response, result: any): void {
```

Comment:
this feels like it should be [middleware](https://expressjs.com/en/guide/writing-middleware.html)

Reply:
Generally, it's not a good idea to add methods that you have to keep calling every time an API call occurs. These should either be refactored away so they're not needed, or made invisible by integrating them as middleware. Pretty much everything else creates tech debt IMO.

Reply:
I'm not familiar enough with middleware to know how to code one up.  I'll look it up and implement it at a later review.  Opened issue #11